homeassistant/components/openuv/.translations/en.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This is incorrect. This is the output of the translation tool. You need to define a `strings.json` in your directory as [per the docs](https://developers.home-assistant.io/docs/en/config_entries_config_flow_handler.html#translations). You can run `script/translations_develop` to generate this file afterwards.

Reply:
I’ve already included `strings.json` in this PR. Are your saying that I should _only_ include that file (and not this one)?

Reply:
Oh, didn't see `strings.json`. Ok, you can leave it .